src/ci/common.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright 2017 Okta, Inc.
+# Copyright 2017-Present Okta, Inc
```

Comment:
🎉

Reply:
lots of copy pasta, i'll fix the others :)